Plugin/clButtonBase.cpp

```diff
@@ -15,6 +15,9 @@
 #define TEXT_SPACER FromDIP(5)
 #else
 #define TEXT_SPACER 5
+#endif
+#if wxCHECK_VERSION(3, 1, 2)
```

Comment:
if you want to write: `if version >= 3.1.2` then you can simply use
```
#if wxVERSION_NUMBER >= 3120
..
#endif
```

Reply:
Changed.

Reply:
How about this:
```c
#if !wxCHECK_VERSION(3, 1, 2)
// ...
#endif
```
rather than:
```c
#if wxCHECK_VERSION(3, 1, 2)
#else
// ...
#endif
```
?

Reply:
Wasn't sure about the ! operator in cpp expressions, but it seems to be working.
Dunno why my change didn't make it here, sorry.